README.md

```
@@ -1,38 +1,95 @@
+```
+2  **MakeMigrations**: This creates database blueprint
+```bash
+python manage.py makemigrations
+```
+3. **CREATE PHYSICAL DATABASE**: Physically build the database structure (db.sqlite3)
+```bash
+python manage.py migrate
+```
+4. **MAKE SERVER LIVE**:  this will give us a url to access the app on browser
 ```bash
 $ python manage.py runserver
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)